### PR TITLE
Login/logout

### PR DESCRIPTION
**Pull request by mirulzaei**

### CODE REVIEW

**Comment by mirulzaei** [comment]
ready

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<%= form_for @user, url: signup_path do |f| %>
+
+        <% if @user.errors.any? %>
```
Comment (ngocvt-0484) [inline app/views/users/_form.html.erb]: tab 2 spaces please

File: app/helpers/users_helper.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module UsersHelper
+  def gravatar_for user 
+    gravatar_id = Digest::MD5::hexdigest user.email.downcase
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
+    image_tag gravatar_url, alt: user.name, class: "gravatar"
+  end
+end
+
```
Comment (ngocvt-0484) [inline app/helpers/users_helper.rb]: remove blank line

**Comment by mirulzaei** [comment]
ready

**Pull request merged**